packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -61,7 +61,7 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
     /// @custom:legacy
     /// @custom:spacer paused
     /// @notice Spacer for backwards compatibility.
-    bool public spacer_53_0_1;
+    bool spacer_53_0_1;
```

Comment:
Nit, looks like we're explicit about visibility elsewhere which I agree with
```suggestion
    bool private spacer_53_0_1;
```

Reply:
agree with making it explicit